client/client_app.py

```
@@ -0,0 +1,168 @@
+                print('data',data)
+                if data['cmd'] == 'start':
+                    print('hi Start')
+                    self.overlay.start_session(data['minutes'])
+                elif data['cmd'] == 'add':
+                    self.overlay.update_session(data['minutes'], add_type=True)
+                elif data['cmd'] == 'sub':
+                    self.overlay.update_session(data['minutes'], add_type=False)
+                elif data['cmd'] == 'end':
+                    self.overlay.end_session()
+                elif data['cmd'] == 'lock':
+                    subprocess.call("rundll32.exe user32.dll,LockWorkStation")
```

Ruff (0.13.1)

Starting a process with a partial executable path

(S607)